spec/models/content_item_spec.rb

```diff
@@ -1,124 +1,75 @@
 require 'rails_helper'
 
 RSpec.describe ContentItem do
-  let(:content_item) do
-    ContentItem.new(
-      content_item_params
-    ).tap do |content_item|
-      content_item.link_set = Tagging::ContentItemExpandedLinks.new(link_set_params)
-    end
-  end
+  subject(:content_item) { described_class.new(data, blacklist: blacklist) }
```

Comment:
Yeah, and I dislike test contexts too :) But that's all a different discussion. We'll have a discussion in a different PR.